src/main/java/org/terasology/launcher/settings/BaseLauncherSettings.java

```diff
@@ -76,49 +72,19 @@
 
     private static final Logger logger = LoggerFactory.getLogger(BaseLauncherSettings.class);
 
-    private static final String COMMENT_SETTINGS = "Terasology Launcher - Settings";
 
     private static final String WARN_MSG_INVALID_VALUE = "Invalid value '{}' for the parameter '{}'!";
     private static final Level LOG_LEVEL_DEFAULT = Level.INFO;
 
-    private final Path launcherSettingsFile;
     private final Properties properties;
 
-    public BaseLauncherSettings(Path launcherDirectory) {
-        launcherSettingsFile = launcherDirectory.resolve(LAUNCHER_SETTINGS_FILE_NAME);
-        properties = new Properties();
-    }
-
-    public String getLauncherSettingsFilePath() {
-        return launcherSettingsFile.toString();
-    }
-
-    @Override
-    public synchronized void load() throws IOException {
-        if (Files.exists(launcherSettingsFile)) {
-            logger.debug("Load the launcher settings from the file '{}'.", launcherSettingsFile);
-
-            // load settings
-            try (InputStream inputStream = Files.newInputStream(launcherSettingsFile)) {
-                properties.load(inputStream);
-            }
-        }
+    BaseLauncherSettings(Properties properties) {
+        this.properties = properties;
     }
 
     @Override
-    public synchronized void store() throws IOException {
-        logger.trace("Store the launcher settings into the file '{}'.", launcherSettingsFile);
-
-        // create directory
-        if (Files.notExists(launcherSettingsFile.getParent())) {
-            Files.createDirectories(launcherSettingsFile.getParent());
-        }
-
-        // store settings
-        try (OutputStream outputStream = Files.newOutputStream(launcherSettingsFile)) {
-            properties.store(outputStream, COMMENT_SETTINGS);
-            logger.trace("Stored settings: {}", this);
-        }
+    public Properties getProperties() {
```

Comment:
Got to expose this so it can be accessed by the `Settings` helper class for writing the properties to file. Scope can be limited to package-private if I'm not mistaken. :monocle_face: